.circleci/config.yml

```diff
@@ -588,6 +587,25 @@ jobs:
       - run: git diff-index --quiet HEAD
       - magma_slack_notify
 
+  # Fail if terraform variables, helm charts and orcl is out of sync
+  deploy-sync-checkin:
+    machine:
+      image: ubuntu-1604:201903-01
+      # docker_layer_caching: true
```

Comment:
Uncomment

Reply:
There was an issue which I uncovered with the tests failing when I uncommented that. I will delete this line to avoid confusion here.

Reply:
Just remove it then